tests/functional/build-hook-ca-fixed.nix

```
@@ -1,6 +1,6 @@
 { busybox }:
 
-with import ./config.nix;
+with import "${builtins.getEnv "_NIX_TEST_BUILD_DIR"}/config.nix";
```

Could this be factored into a file?
I think this would be nice:

```suggestion
with import ./config.nix;
```

\+

`tests/functional/config.nix`:
```nix
import "${builtins.getEnv "_NIX_TEST_BUILD_DIR"}/config.nix"
```

---

Oh hah! that's not bad :)

---

Oh but let's do that after we get rid of the old build system, because it will probably break it when we have a `config.nix.in` and `config.nix` in the same dir.